versions/views.py

```diff
@@ -154,3 +155,25 @@ def get_object(self, queryset=None):
             return Version.objects.most_recent()
 
         return get_object_or_404(Version, slug=version_slug)
+
+
+class InProgressReleaseNotesView(TemplateView):
+    template_name = "versions/in_progress_release_notes.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context.update(
+            {
+                "release_notes": self.get_in_progress_release_notes(),
+            }
+        )
```

Comment:
I think I would just use the simpler

```suggestion
        context["release_notes"] = self.get_in_progress_release_notes()
```